app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,170 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+
+body {
+  background-color: #ebe7df;
+  color: #333333;
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+}
+
+.row h1 {
+  background: #418a95;
+  color: #fff;
+  font-weight: bold;
```

Comment:
Properties should be ordered background, color, font-size, font-weight, margin-bottom, padding